provider/kubernetes/kubernetes.go

```diff
@@ -186,6 +186,17 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 				witelistSourceRangeAnnotation := i.Annotations[annotationKubernetesWhitelistSourceRange]
 				whitelistSourceRange := provider.SplitAndTrimString(witelistSourceRangeAnnotation)
 
+				priority := len(pa.Path)
+
+				priorityString, ok := i.Annotations[types.LabelFrontendPriority]
+				if ok {
+					priorityParsed, err := strconv.Atoi(priorityString)
+
+					if err == nil {
```

Comment:
Actually in current logic it will fall back to the default priority of len(Path), as if no priority is set.

Personally I think that dropping the frontend is ok when it has wrong Auth (because we obviously don't wanna show something secret to all users), but here probably routing with default priority is fine.